apps/academy/src/pages/index.tsx

```diff
@@ -13,7 +13,7 @@ const Home: NextPageWithLayout = () => {
           <div className="hidden justify-center md:flex">
             <div className="text-bttf-lg">
               <h1>learn</h1>
-              <h1>web3 __</h1>
```

Comment:
We are getting a bit further away from the original design, are we not? It seems the original isn't actually possible to add an extra space between 'WEB' and the underscore. The 'H' in 'with' is now looking like an 'A', or maybe an 'R'. When I land on the page, I know it says 'learn web3 with friends', but I (getting subjective here) am constantly drawn back to it to question which letter it is. A bit of unnecessary distraction. 
![learn0](https://github.com/Developer-DAO/academy-turbo/assets/40469149/116c94f1-d3b7-4fcc-b2bf-e3110ce4a6a9)
![learn1](https://github.com/Developer-DAO/academy-turbo/assets/40469149/a1fa2165-581e-4c45-8e0e-becf271e6b8e)
![learn2](https://github.com/Developer-DAO/academy-turbo/assets/40469149/b9862b60-3068-44e2-a55f-3b14002ba6f8)

Reply:
Personally, I think we should 'let it pass' for launch, but we should come back to it. I think we'll have to do that anyway.

Reply:
agree theres rough edges here. possibly could be remedied by just need a tweak to the line height here, but the underscores do dip below the line (by definition). ill see if a small change is worthwhile now.